### PR TITLE
Fix bug 1541252 - Fixes all build errors on Python 3

### PR DESCRIPTION
**Pull request by jotes**
TODO:
* [x] - pylama lints skipped files

### CODE REVIEW

**Comment by jotes** [comment]
Hi,
I upgraded `pylama` to the newest version because it supports Python 3.
Unfortunately, the upgrade breaks the existing behaviour of pylama, because `skip` option is omitted.
This produces a lot of noise because it displays error messages for e.g. database migrations.
There's the pull request in the upstream: https://github.com/klen/pylama/pull/142/commits
However, I don't know when the fix will be released.

@adngdb @mathjazz I want to check if `flake8` will be the good replacement for `pylama`. Do you have any thoughts/ideas/preferences about linting-tools?

**Comment by adngdb** [comment]
It seems `pylama` does a bunch more things than just calling `flake8`, but I also don't care too much. If using `flake8` directly unblocks you, then I'm OK with it.

**Comment by jotes** [comment]
@adngdb Hey, could you look at this again? I've updated the commit message (feel free to ask if something is not clear).

**Comment by jotes** [comment]
@adngdb that's right

File: Makefile [hunk 1]
```diff
@@ -41,6 +42,13 @@ build:
 build-py3: override PYTHON_VERSION=3.7.3
 build-py3: build
 
+test-py3: override PYTHON_VERSION=3.7.3
+test-py3: test
+
+shell-py3: override PYTHON_VERSION=3.7.3
```
[the remark this answers is not in the document]
Comment (jotes) [reply Makefile]: The variable is defined here: https://github.com/mozilla/pontoon/pull/1288/files#diff-b67911656ef5d18c4ae36cb6741b7965L3. When the bump will happen, we'll have to remove all `*-py3` targets.

[the remark this answers is not in the document]
Comment (adngdb) [reply Makefile]: Ah, OK, that makes sense. Sorry! :)

**Pull request merged**